app/extensions/brave/content/styles/defaultStyles.css

```diff
@@ -0,0 +1,18 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
future refactoring of about pages will benefit from this instead of relying on `common.less` file so I took advantage of this PR to add it